app/src/androidTest/java/com/github/sdp_begreen/begreen/activities/MainActivityTest.kt

```diff
@@ -273,7 +272,31 @@ class MainActivityTest {
     }
 
     @Test
-    fun testContactUsBottomSheetMessageCorrected() {
+    fun cancelButtonCorrectlyHidesTheBottomSheet() {
```

Comment:
In the name what is the bottom sheet? I'm not sure of that name

Reply:
This name is not very self-explanatory, I admit it. I changed `BottomSheet` with `ContactUsWindow`, what do you think about it?